omnibus/.kitchen.yml

```diff
@@ -14,6 +14,8 @@ provisioner:
 platforms:
   - name: ubuntu-12.04
     run_list: apt::default
+  - name: ubuntu-16.04
```

Comment:
Should there also be a 16.04 entry in [`omnibus/cookbooks/omnibus-supermarket/.kitchen.yml`](https://github.com/chef/supermarket/blob/2.7.4/omnibus/cookbooks/omnibus-supermarket/.kitchen.yml#L15-L16) for testing a 16.04 build locally?